src/t2fs.c

```diff
@@ -43,10 +43,33 @@ FILE2 create2 (char *filename);
 
 int delete2 (char *filename);
     
-FILE2 open2 (char *filename);
+FILE2 open2 (char *filename) {
 
 
-int close2 (FILE2 handle);
+    struct t2fs_record* entrada = malloc(sizeof(struct t2fs_record));
```

Comment:
SearchEntradas ja devolve um ponteiro
```suggestion
    struct t2fs_record* entrada = malloc(sizeof(struct t2fs_record));
```

Reply:
alocado